src/index.html

```diff
@@ -11,7 +11,209 @@
+            </label>
+            <input
+              type="email"
+              name="mail"
+              id="mail"
+              placeholder="email@example.com"
+              required
+              multiple
+            >
+          </div>
+          <div class="form-field">
+            <label for="password">
+              Password:
+            </label>
+            <input
+              type="password"
+              name="password"
+              id="password"
+            >
+          </div>
+        </fieldset>
+      </div>
+      <div class="facts zone">
+        <fieldset>
+          <legend>
+            An interesting fact about you!
+          </legend>
+          <div class="form-field">
+            Do you love cats?
+            <label for="Y">
+              <input
+                type="radio"
+                name="yesOrNo"
+                id="Y"
+                value="Y"
+              >
+              Yes
+            </label>
+            <label for="N">
+              <input
+                type="radio"
+                name="yesOrNo"
+                id="N"
+                value="N"
+              >
+              No
+            </label>
+          </div>
+          <div class="form-field">
+            <label for="colors">
+              What is your favorite color?
+            </label>
+            <input
+              type="color"
+              name="colors"
+              id="colors"
+            >
+          </div>
+          <div class="form-field">
+            <label for="time">
+              What time do you go to bed?
+            </label>
+            <input
+              type="time"
+              name="time"
+              id="time"
+            >
+          </div>
+          <div class="form-field">
+            <label for="car">
+              What are your favorite brand of cars?
+              <select
+                class="choose"
+                name="car"
+                id="car"
+                multiple
+              >
+                <option value="bmw">BMW</option>
+                <option value="audi">Audi</option>
+                <option value="lada">Lada</option>
+              </select>
+            </label>
+          </div>
+          <div class="form-field">
+            <label for="rate">
+              How do you rate our work?
+            </label>
+            <input
+              type="range"
+              name="rate"
+              id="rate"
+              value="0"
+            >
+          </div>
+        </fieldset>
+      </div>
+      <div class="additionalInfo zone">
+        <fieldset>
+          <legend>
+            Additional info:
+          </legend>
+          <div class="form-field">
+            <label for="comments">
+              Comments:
+              <textarea
+                name="comments"
+                id="comments"
+                cols="18"
+                rows="2"
+              >
+              </textarea>
```

Comment:
```suggestion
              ></textarea>
```
<img width="300" alt="image" src="https://user-images.githubusercontent.com/53611910/208320861-3ca2e996-ea39-4428-ac05-d45066b5b58b.png">